sdk/typescript/src/dwallet-mpc/import-dwallet.ts

```diff
@@ -34,13 +34,17 @@ function isNewImportedKeyDWalletEvent(event: any): event is NewImportedKeyDWalle
 	return event.dwallet_id !== undefined && event.dwallet_cap_id !== undefined;
 }
 
-export async function createImportedDWallet(conf: Config): Promise<DWallet> {
+export async function createImportedDWallet(
+	conf: Config,
+	secretShare: Uint8Array,
```

Comment:
So this is not the best API, a better idea is to take the seed and generate the share to import, but its fine for now - let's just remember this for when we are doing the typescript refactors @zeevmoney. @ItayLevyOfficial can you add it to an issue if one exists, or open a new one if it doesn't?